msteams-platform/concepts/app-fundamentals-overview.md

```diff
@@ -66,20 +66,29 @@ Understanding the user and their concern are the first indicators of how a Teams
 
 Government Community Cloud (GCC) is a government focused copy of the commercial environment. Department of Defense (DOD) and Federal contractors must meet the stringent cybersecurity and compliance requirements. For this purpose, GCC-High was created to meet the needs of DOD and Federal contractors. GCC-High is a copy of the DOD cloud but exists in its own sovereign environment. The DOD cloud is built for the Department of Defense only.
 
+Endpoints for government cloud are:
+
+| Tenant | GCC | GCC-High | DOD |
+|-------------|---------|---|---|
+|Teams client|`https://teams.microsoft.com`|`https://gov.teams.microsoft.us/`|`https://dod.teams.microsoft.us/` |
+|Teams admin |`https://admin.teams.microsoft.com/`|`https://admin.gov.teams.microsoft.us/`|`https://admin.dod.teams.microsoft.us`|
+|Microsoft Graph |`https://graph.microsoft.com`|`https://graph.microsoft.us`|`https://dod-graph.microsoft.us`|
+
 The following table includes Teams features and availability for GCC, GCC-High, and DOD:
 
 | Features   | GCC | GCC-High | DOD |
 |-------------|---------|---|---|
 | Teams owned apps as in internally developed apps | ✔️ App is enabled if it has GCC | ✔️ App is enabled if it has GCC-High | ✔️ App is enabled if it has DOD |
 | Microsoft apps | ✔️ Microsoft apps compliant with GCC | ✔️ Microsoft apps compliant with GCC-High | ✔️ Microsoft apps compliant with DOD |
 | 3P or third-party apps | ✔️ Third-party apps are available. Disabled by default and tenant admin use their own discretion to enable it. | ❌ | ❌ |
-| Bots | ✔️ | ❌ | ❌ |
-| Custom or Lob tab apps |  ✔️ | ✔️ | ✔️ |
+| Custom or Lob tab apps |  ✔️ | ✔️(****Compliance UI***) | ✔️(****Compliance UI***) |
+| Custom or Lob bots | ✔️ | ✔️(****Compliance UI***) | ❌ |
+| Custom message extensions | ✔️ | ✔️ | ❌ |
 | Sideloading apps | ✔️ | ❌ | ❌ |
-| Custom or Lob bots | ✔️ | ❌ | ❌ |
-| Custom message extensions | ❌ | ❌ | ❌ |
 | Custom connectors | ❌ | ❌ | ❌ |
 
+****Compliance UI***: By enabling third-party communications, customers accept that such communication is being processed through the third party and not Microsoft. The customer is solely responsible for mitigating risks associated with connecting with third party bots in their services. Microsoft does not endorse and makes no warranties, express, or implied concerning the security of third parties the customer allows to connect with their service. Enabling bots will extend your system boundary beyond this tenant based on the bot you choose to leverage. It is your responsibility to ensure that this meets your compliance requirements including FedRAMP, DFARS, ITAR, etc. It is your responsibility to evaluate the risk and compliance of any endpoint and URL that you connect to.
```

Comment:
in their "services" - what services are we referring to? Did we mean tenant? 

And I think this might be an app-level consent and not just specific to a bot. Imagine an app connecting to non-us endpoints through personal/shared tabs.

Reply:
We are specific that this is only applicable for GCCH bots. Are you saying that in GCCH they can use a bot that an app can connect to a non-us endpoint?